src/FileConventions/Library.fs

```diff
@@ -329,3 +329,46 @@ let DetectInconsistentVersionsInFSharpScripts
         false
     else
         DetectInconsistentVersionsInNugetRefsInFSharpScripts fsxFiles
+
+let allowedNonVerboseFlags =
+    seq {
+        "unzip"
+
+        // even if env in linux has --split-string=foo as equivalent to -S, it
+        // doesn't seem to be present in macOS' env man page and doesn't work either
+        "env -S"
+    }
+
+let NonVerboseFlagsInGitHubCI(fileInfo: FileInfo) =
+    let validExtensions =
+        seq {
+            ".yml"
+            ".fsx"
+            ".fs"
+            ".sh"
+        }
+
+    assert
+        validExtensions
+        |> Seq.map(fun ext -> fileInfo.FullName.EndsWith(ext))
+        |> Seq.contains true
+
+    let fileLines = File.ReadLines(fileInfo.FullName)
```

Comment:
@realmarv nit: no need for parentheses in lines 353 and 356

Reply:
Done